pkg/wireguard/conf.go

```diff
@@ -146,13 +153,11 @@ func (d DNSOrIP) String() string {
 func Parse(buf []byte) *Conf {
```

Comment:
After using ParseDump in applyTopology, we don't need the Parse function anymore, right?

Reply:
Same for the ShowConf being replaced by ShowDump.

Reply:
ok, I just saw, that you are using it to test the ParseDump() function. That's cool.